osi_object.proto

```diff
@@ -54,6 +54,15 @@ message Vehicle
     /// The id of the driver of the (ego) vehicle.
     /// Field need not be set if ego_vehicle is set to false.
     optional Identifier driver = 10;
+    
+    /// The vector pointing from the bounding box center point (base.position) to the middle (in x, y and z) 
+    /// of the rear axle under neutral load conditions.
+    optional Vector3d bbcenter_to_rear = 11;
+
+	/// The vector pointing from the bounding box center point (base.position) to the middle (in x, y and z) 
+	/// of the front axle under neutral load conditions.    		
+	optional Vector3d bbcenter_to_front = 12;
+    
 
```

Comment:
One more question: in which coordinate system was relative_ego_origin defined? In world coordinates? Then it has to be updated each frame. Or in object coordinates (then it would be constant)? That would be consistent with our previous discussion, about the necessity to rotate relative_ego_origin from object to world frame.
In case we have an object frame, we should also define it in the wiki.